README.md

```diff
@@ -164,14 +153,15 @@ ops version
 ```
 
 get the release channel (or nightly)
+
 ```sh
 ls .ops/
 ```
```

Comment:
I didn't quite understood how this command is related to the text above it?

Reply:
The ~/.ops directory contains one subfolder for each Nanos release, so by looking at the contents of that directory it is possible to know what Nanos release is being used by ops. But nowadays the `ops version` command outputs not only the version of ops itself, but the Nanos version as well, so I think we can remove this part in the README file.

Reply:
Than you for the exploitation, I have removed it.